src/pages/PlayPage.js

```diff
@@ -22,11 +23,17 @@ export default function PlayPage({
 
   return (
     <Container>
-      {playing ? (
-        duration && <Timer duration={duration} />
-      ) : (
-        <PlayButton onClick={onPlay}>&gt;</PlayButton>
-      )}
+      <Wrapper>
+        {playing ? (
```

Comment:
The same kind of thing you call `isPlaying` at some other place. Why not here?

Reply:
I renamed it in the App.js and then forgot about the play page...